Updated pre-requisite for from.email (#81292)
Azure/azure-content: articles/active-directory-b2c/custom-email-sendgrid.md

Add the following claims transformation to the `<ClaimsTransformations>` element within `<BuildingBlocks>`. Make the following updates to the claims transformation XML:

* Update the `template_id` InputParameter value with the ID of the SendGrid transactional template you created earlier in [Create SendGrid template](#create-sendgrid-template).

> * Update the `from.email` address value. Use a valid email address to help prevent the verification email from being marked as spam. **NOTE**: The email address that you are going to use here needs to verified in Send Grid under sender authentication either with Domain Authentication or with Single Sender Authentication.

Reviewer: ```suggestion
* Update the `from.email` address value. Use a valid email address to help prevent the verification email from being marked as spam. 
   > [!NOTE]
   > This email address must be verified in SendGrid under Sender Authentication with either domain authentication or Single Sender Authentication.
```
